docs/general/builtins/scalar.rst

```diff
@@ -805,6 +805,29 @@ The following fields are supported:
 .. _`available time zones`: http://www.joda.org/joda-time/timezones.html
 .. _`Joda-Time`: http://www.joda.org/joda-time/
 
+.. _current_time:
+
+``CURRENT_TIME``
+----------------
+
+The ``CURRENT_TIME`` expression returns the time in microseconds
+since midnight UTC at the time the SQL statement was handled. Therefore,
```

Comment:
Reading this again I think the `Therefore, ...` implies that the prior sentence explains the behavior of `the same time value is returned for every invocation within a single
statement.` but there is no logical correlation between the two.

The first statement would actually imply that there could be different values.

I think this should be rephrased a bit to instead point out that the clock time is looked up at most once within the scope of a single query to ensure multiple `current_time` occurences evaluate to the same value.